README.md

```diff
@@ -100,14 +101,14 @@ All commands should work for at least git version 2.13.0. See the [git website](
     - [Recover a deleted tag](#recover-a-deleted-tag)
     - [Deleted Patch](#deleted-patch)
     - [Exporting a repository as a Zip file](#exporting-a-repository-as-a-zip-file)
-    - [Push a branch and tag that have the same name](#push-a-branch-and-a-tag-that-have-the-same-name)
+    - [Push a branch and a tag that have the same name](#push-a-branch-and-a-tag-that-have-the-same-name)
   - [Tracking Files](#tracking-files)
     - [I want to change a file name's capitalization, without changing the contents of the file](#i-want-to-change-a-file-names-capitalization-without-changing-the-contents-of-the-file)
     - [I want to overwrite local files when doing a git pull](#i-want-to-overwrite-local-files-when-doing-a-git-pull)
     - [I want to remove a file from Git but keep the file](#i-want-to-remove-a-file-from-git-but-keep-the-file)
     - [I want to revert a file to a specific revision](#i-want-to-revert-a-file-to-a-specific-revision)
     - [I want to list changes of a specific file between commits or branches](#i-want-to-list-changes-of-a-specific-file-between-commits-or-branches)
-    - [I want Git to ignore changes to a file without deleting it](#i-want-git-to-ignore-changes-to-a-specific-file)
+    - [I want Git to ignore changes to a specific file](#i-want-git-to-ignore-changes-to-a-specific-file)
```

Comment:
Was this supposed to happen? Did you use the automated doctoc?

Reply:
This seemed to have happened after I ran the doctoc after I added my initial changes. I thought the current branch might had an outdated TOC anyways.

Reply:
Just re-ran the doctoc against master without any of my changes and the following two lines from TOC seem to be outdated:

```
-    - [Push a branch and tag that have the same name](#push-a-branch-and-a-tag-that-have-the-same-name)
+    - [Push a branch and a tag that have the same name](#push-a-branch-and-a-tag-that-have-the-same-name)

-    - [I want Git to ignore changes to a file without deleting it](#i-want-git-to-ignore-changes-to-a-specific-file)
+    - [I want Git to ignore changes to a specific file](#i-want-git-to-ignore-changes-to-a-specific-file)
```

Reply:
I really don't mind. All good. :)